installer/conf/sudoers

```diff
@@ -31,6 +31,9 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 #run hdinsightmanifestreader.rb
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsoft/omsagent/bin/hdinsightmanifestreader.rb
 
+# Enable nxOMSAuditdResource
+omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSAuditdPlugin.sh
```

Comment:
OMSAuditdPlugin.sh is NOT a bundle! It is the script used by the nxOMSAUditdPlugin DSC resource to complete the enable/disable "SET" operation.